### ISSUE TITLE
Server not receiving (or sending? )info from player connected after respawn

### ISSUE THREAD

**Issue by artur9010** [Pryaxis/TShock#1771]
* TShock version: 4.4.0 Pre 1

#### Reproduction steps (if applicable)?

If I die then server will not register anything I do after respawn (everyone esle sees me standing in place and not doing enything). Other players deaths works fine.

I can see other players moving just fine, but if I go to not yet loaded part of the map then no blocks will load and I will fall trough the world

Reconnecting fixes the issue until next respawn.

#### Any stack traces or error messages (if known)?
Nothing.

#### Any log messages from files that end in `.log`?
Also nothing.

#### What plugins and what versions of those plugins are you running?
None

**Comment by kubedzero** [comment]
possibly related to https://github.com/Pryaxis/TShock/issues/1779 but I think I also experienced this. Dying and respawning did not seem to fix the issue, but I did not yet try a disconnect and reconnect from the server.  I also did not try the block loading part but will try that next time. 

I can confirm though that there were not any logs relevant, only player connects and disconnects at the beginning and ending of the game session
